### PR TITLE
Add avatar

### PR DESCRIPTION
**Pull request by tervay**
image

image

### CODE REVIEW

File: pwa/app/components/tba/teamAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      alt="Team Avatar"
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      src={`data:image/png;base64, ${media.details['base64Image']}`}
+      className={cn(
+        'size-12 rounded inline mr-2 p-1 cursor-pointer',
+        colorClass,
+      )}
+      onClick={() => {
+        if (colorClass === 'bg-blue-500') {
```
Comment (fangeugene) [inline pwa/app/components/tba/teamAvatar.tsx:21]: Can we use the official FIRST red/blue values that they specify for the audience display?

Comment (tervay) [reply pwa/app/components/tba/teamAvatar.tsx]: Yeah, do you know where I can find them?

File: pwa/.eslintrc.cjs [hunk 2]
```diff
@@ -54,6 +54,9 @@ module.exports = {
           typescript: {},
         },
       },
+      rules: {
+        'jsx-a11y/no-noninteractive-element-interactions': 'off',
```
Comment (tervay) [inline pwa/.eslintrc.cjs:58]: I simply don't have the patience to do this right now

File: pwa/app/components/tba/teamAvatar.tsx (hunk 1 above)
Comment (fangeugene) [reply pwa/app/components/tba/teamAvatar.tsx]: https://github.com/the-blue-alliance/the-blue-alliance/pull/6020/files

**Pull request merged**